VKappProject/VKappProject/Controllers/FriendCollectionViewController.swift

```diff
@@ -0,0 +1,29 @@
+// FriendCollectionViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Ячейка с фотографией друга
```

Comment:
```suggestion
/// Ячейка с фотографией друга
```
Контролер это а не ячейка